src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -42,7 +42,10 @@ function FrameLocation({ frame }: FrameLocationProps) {
 
   const filename = getFilename(frame.source);
   return (
-    <div className="location">{`${filename}: ${frame.location.line}`}</div>
+    <div className="location" title={"Line number:" + frame.location.line}>
+      <div className="fileName">{filename}: </div>
+      <div className="lineNumber">{frame.location.line}</div>
```

Comment:
do we want separate `<div>`s or spans?

Reply:
Spans :). Still working the width issue with the tabs. Would you be against adding state to the Tab component? I haven't found a way of implementing dynamic resizing that isn't convoluted and that goes with best-practices.

Reply:
sure, give it a try!